sensor_msgs/PointCloud2 point_cloud
float32 leaf_size
---
sensor_msgs/PointCloud2 voxelized_point_cloud